### PR TITLE
Adds nightly e2e for hiv prevalence-diagnoses-deaths

### LINKED ISSUE #2547: Nightly e2e coverage for  hiv prevalence
Nightly e2e coverage for  hiv prevalence

### PR DESCRIPTION
**Pull request by JDemlow**
# Description and Motivation
Closes #2547, #2548, #2549
Adds hiv prevalence, diagnosis, and deaths to one test.
Also renames asthma test to remove `.ci`

## Has this been tested? How?

Using vscode terminal.

hiv-prevalence-diagnoses-deaths

## Types of changes

(leave all that apply)

- New content or feature

### CODE REVIEW

File: frontend/playwright-tests/asthma.spec.ts
Comment (eriwarr) [inline frontend/playwright-tests/asthma.spec.ts:1]: Why remove <sub>.ci</sub> from the test name?

Comment (JDemlow) [reply frontend/playwright-tests/asthma.spec.ts:1]: In the open source channel on slack Ben mentioned that all the new per topic tests will just be “nightly” and not include “ci” in the file name. So I thought the last PR I did needed to have the .ci removed

Comment (eriwarr) [reply frontend/playwright-tests/asthma.spec.ts:1]: Ah, gotcha! Can you update it to `asthma.nightly.spec.ts`?

Comment (JDemlow) [reply frontend/playwright-tests/asthma.spec.ts:1]: Oh yeah! I see that now. I'm on it

Comment (JDemlow) [reply frontend/playwright-tests/asthma.spec.ts:1]: Done!

**Review by eriwarr — APPROVED** [review]
Awesome work! Thanks, Jim!

**Pull request merged**